src/community/index.md

```diff
@@ -8,17 +8,17 @@ layout: page
 
 A-Frame grown into a large active community by putting VR content creation into everyone's hands.
 
-* [A-Frame GitHub Repo](https://github.com/aframevr/aframe/) - Grab the code, file issues, and contribute.
+* [A-Frame GitHub Repo](https://github.com/aframevr/aframe) - Grab the code, file issues, and contribute.
 * A-Frame Community Slack Channel - [Sign up](https://aframevr-slack.herokuapp.com/) to share your work and join the discussion.
 * [A-Frame Twitter](https://twitter.com/aframevr) - Tweet at us.
-* [**Made With A-Frame** Tumblr](http://aframevr.tumblr.com/) - Check out a showcase of recent work done in A-Frame.
-* [**awesome-aframe** Repo](https://github.com/aframevr/awesome-aframe) - Go over a list of awesome things created regarding A-Frame.
-* [Stack Overflow](http://stackoverflow.com/questions/tagged/aframe) - Ask questions and learn on Stack Overflow.
+* [A-Frame Blog](https://aframe.io/blog/) - Stay up to date with the latest A-Frame changes and projects.
+* [**awesome-aframe** Repo](https://github.com/aframevr/awesome-aframe) - Peruse a list of awesome things created for or using A-Frame.
+* [Stack Overflow](https://stackoverflow.com/questions/tagged/aframe) - Ask questions and learn on Stack Overflow.
 
 ## WebVR
 
 WebVR is not just A-Frame. See what is happening in the greater community.
 
-* [WebVR Community Slack Channel](http://webvr.slack.com/) - [Sign up](https://webvr-slack.herokuapp.com/).
+* WebVR Community Slack Channel - [Sign up](https://webvr-slack.herokuapp.com/) to join the WebVR community on Slack.
```

Comment:
The last part is a bit redundant

WebVR Community Slack Channel ... WebVR community on Slack